uber/templates/registration/form.html

```diff
@@ -51,7 +49,6 @@
 {{ attendee_fields.emergency_contact }}
 {{ attendee_fields.cellphone }}
 {{ attendee_fields.address }}
-{{ attendee_fields.amount_extra }}
```

Comment:
The very first thing I did when investigating https://github.com/magfest/ubersystem/issues/3472 was to look for the kick-in level near the payment fields on the admin form. Scrolling back up and finding it among the other fields just felt awkward, so I moved it back.